routes/web.php

```diff
@@ -14,6 +14,13 @@
     return redirect('/login');
 });
 
+Route::get('/record/create', 'RecordController@create')->name("record.create");
+Route::post('/record/save', 'RecordController@save')->name("record.save");
+Route::get('/record/sort/{order}','RecordController@sort')->name("record.sort");
+Route::get('/record/list', 'RecordController@list')->name("record.list");
+Route::get('/record/delete/{id}', 'RecordController@delete')->name("record.delete"); //TODO: Change to Delete (Crashes).
```

Comment:
Put post, to solve the problem you have to change the form with the method delete